.github/workflows/cd.main.yml

```diff
@@ -0,0 +1,59 @@
+name: CD
+on:
+  push:
+    branches:
+      - main
+env:
+  ORGANIZATION_SLUG: templates
+  INSTRUQT_TOKEN: ${{ secrets.INSTRUQT_TOKEN }}
+jobs:
+  deploy-changes:
+    name: Deploy changes
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install CLI
+        run: |
+          curl https://github.com/instruqt/cli/releases/download/1909-0b7cb45/instruqt-linux-1909-0b7cb45.zip -L -o cli.zip
```

Comment:
Is there a consistent way to download the latest version? Not sure how our our `docker/cli` container can help us here, but that would be one way.

Reply:
There is a docker container with the CLI pre installed.

Reply:
Perhaps I am missing something, but I don't think we can use this as a base image for GitHub actions. The image has no shell installed it would seem to run shell scrips, and it also doesn't copy over local files.

If this is indeed possible, can you show me? I haven't found a good way to do this.